server/errors.ts

```diff
@@ -62,17 +69,19 @@ class NotImplementedError extends AppError
 
 //----------------------------------------------------------------------------------------------------------------------
 
-class DuplicateSupplementError extends AppError
+export class DuplicateSupplementError extends AppError
```

Comment:
Looks like a legacy bit, but...

Seems odd that the duplicate error makes explicit mention of supplements, when the other errors look pretty generic.

Reply:
It is, and it's an intentionally specific error. But sure, it could become a `DuplicateItemError`, I guess.